cvprac/cvp_client.py

```diff
@@ -661,34 +661,50 @@ def _make_request(self, req_type, url, timeout, data=None,
                     raise error
                 continue
             break
-        resp_data = None
```

Comment:
We do not need the initialization because every block returns. The only way we get to the return on 708 is if the line 677 is successful and doesn't hit the /resource/ API if statement.